README.md

```diff
@@ -24,17 +24,14 @@ their work.
 
 ## Prerequisites
 
-Using this project and running its components requires
-[Node 8](https://nodejs.org/) and
+This project requires [Node 8](https://nodejs.org/) and
 [Docker](https://www.docker.com/community-edition).
 
-( ◡‿◡ *)
```

Comment:
rip kaomoji

Reply:
Sadly, one too many. Meant to fit it in somewhere else. `/code/part-two/README.md` is bizarrely lacking in kaomoji. Someone should submit a GitHub Issue.